cloudbuild.Dockerfile

```diff
@@ -1,5 +1,6 @@
 # The image has the environment setup for running builds and tests
```

Comment:
Is there anything specific from gcr.io/cloud-builders/go we were using? If it's fine to use generic golang image, we should remove the comment in lines 1-2.